source/configuration_manual/lazy_expunge_plugin.rst

```diff
@@ -72,132 +110,113 @@ Example configuration:
-    prefix =
-    separator = /
-    inbox = yes
-  }
-
-  # namespaces for lazy_expunge plugin:
-  namespace {
-    prefix = .EXPUNGED/
-    separator = /
-    location = maildir:~/Maildir/expunged
-  }
-  namespace {
-    prefix = .DELETED/
-    separator = /
-    location = maildir:~/Maildir/deleted
-  }
-  namespace {
-    prefix = .DELETED/.EXPUNGED/
-    separator = /
-    location = maildir:~/Maildir/deleted/expunged
-  }
-
-  mail_plugins = $mail_plugins lazy_expunge
-  plugin {
-    lazy_expunge = .EXPUNGED/ .DELETED/ .DELETED/.EXPUNGED/
-  }
-
-Multi-dbox
-==========
-
-With multi-dbox use different MAILBOXDIRs (so copying between namespaces works
-quickly within the same storage), but otherwise exactly the same paths (index,
-control):
-
-.. code-block:: none
-
-  # the default namespace
+  # Default namespace
   namespace {
     prefix =
     inbox = yes
     location = mdbox:~/mdbox:INDEX=/var/index/%d/%n
+    separator = /
   }
 
-  # lazy_expunge namespace(s):
+  # lazy_expunge namespace(s)
   namespace {
     prefix = .EXPUNGED/
     hidden = yes
     list = no
+    separator = /
     subscriptions = no
 
     location = mdbox:~/mdbox:INDEX=/var/index/%d/%n:MAILBOXDIR=expunged
-    # If mailbox_list_index=yes is enabled, it needs a separate index file (v2.2.28+):
+
+    # If mailbox_list_index=yes is enabled, it needs a separate index file
+    # (v2.2.28+):
     #location = mdbox:~/mdbox:INDEX=/var/index/%d/%n:MAILBOXDIR=expunged:LISTINDEX=expunged.list.index
   }
 
 Copy only the last instance (v2.2+)
-===================================
+-----------------------------------
+
+If a mail has multiple copies within a user account, each copy is normally
+moved to the lazy expunge storage when it's expunged.
+
+Example: this may happen when moving a message to Trash, as clients can issue
+IMAP COPY command to copy the message to Trash before expunging the message
+from the original mailbox.  Deleting later from Trash would result in two
+copies of the same message in the lazy expunge storage.
+
+With v2.2+ you can set ``lazy_expunge_only_last_instance = yes`` to copy only
+the last instance to the expunge storage.  This ensures that only a single
+copy of a message will appear in the expunge storage.
 
-If mail has multiple copies (via IMAP COPY), each copy is normally moved to
-lazy expunge namespace when it's expunged. With v2.2+ you can set ``plugin {
-lazy_expunge_only_last_instance=yes }``  to copy only the last instance and
-immediately expunge the others. This may be useful if you want to provide a
-flat list of all expunged mails without duplicates in your webmail. With many
-clients this means that the last instance is always in the Trash mailbox.
+Quota
+-----
+
+Generally, it is desired that messages in expunge storage are NOT
+NOT counted towards user quota, as the messages seen by the user will not
+match-up with the size of the quota otherwise (especially if expunge storage
+is hidden from users via ACL).
```

Comment:
:ref:plugin-acl or so?

Reply:
Sure, although I added the reference to the first mention of ACL on the page (a few lines above)